backends/p4tools/modules/smith/common/statements.cpp

```diff
@@ -28,12 +29,12 @@ IR::Statement *StatementGenerator::genStatement(bool is_in_func) {
     if (is_in_func) {
         pctExit = 0;
     }
-    std::vector<int64_t> percent = {PCT.STATEMENT_SWITCH,
-                                    PCT.STATEMENT_ASSIGNMENTORMETHODCALL,
-                                    PCT.STATEMENT_IF,
-                                    PCT.STATEMENT_RETURN,
-                                    pctExit,
-                                    PCT.STATEMENT_BLOCK};
+    std::vector<int64_t> percent = {PCT.STATEMENT_SWITCH, PCT.STATEMENT_ASSIGNMENTORMETHODCALL,
+                                    PCT.STATEMENT_IF, PCT.STATEMENT_RETURN, pctExit,
+                                    PCT.STATEMENT_BLOCK,
+                                    // Add the for-loop statement and
+                                    // the for-in-loop statement generation percentages.
+                                    PCT.STATEMENT_FOR, PCT.STATEMENT_FOR_IN};
```

Comment:
Add a comma here to fix the formatting. 
```suggestion
                                    PCT.STATEMENT_FOR, PCT.STATEMENT_FOR_IN,};
```